app/src/main/res/values/styles.xml

```diff
@@ -124,24 +131,102 @@
         <item name="colorControlHighlight">@color/dark_ripple_color</item>
 
         <item name="preferenceTheme">@style/PreferenceThemeOverlay.v14.Material</item>
-    </style>
 
-    <style name="DarkTheme.Switchable">
-        <item name="android:windowAnimationStyle">@style/SwitchAnimation</item>
     </style>
 
     <style name="BlackTheme" parent="DarkTheme">
+        <item name="colorPrimary">@color/black</item>
+        <item name="colorPrimaryDark">@color/black</item>
+        <item name="colorAccent">@color/white</item>
+        <item name="theme_view_color">@color/white</item>
+        <item name="white_green_text_color">@color/white</item>
+
+        <item name="android:editTextColor">@color/white</item>
+        <item name="android:textColorPrimary">@color/white</item>
+        <item name="android:textColorSecondary">@color/white</item>
+        <item name="android:textColorTertiary">@color/white</item>
+
         <item name="android:windowBackground">@color/black_background_color</item>
         <item name="windowBackground">@color/black_background_color</item>
-
         <item name="separator_color">@color/black_separator_color</item>
         <item name="contrast_background_color">@color/black_contrast_background_color</item>
+
+
+        <item name="checked_selector_drawable">@drawable/dark_checked_selector</item>
+        <item name="queue_background_color">@color/dark_queue_background_color</item>
+        <item name="toolbar_shadow_drawable">@drawable/toolbar_shadow_dark</item>
+        <item name="selector_drawable">@drawable/dark_selector</item>
+        <item name="colorControlHighlight">@color/dark_ripple_color</item>
+
+        <item name="preferenceTheme">@style/PreferenceThemeOverlay.v14.Material</item>
+    </style>
+
+    <style name="TerminalTheme" parent="BlackTheme">
```

Comment:
At the moment we have an inconsistent naming of the resources. While the theme is called `TerminalTheme` we use `hacking` in the names of the resources. Please rename the them to terminal.